src/sdg_hub/flows/qa_generation/document_grounded_qa/enchance_multi_summary_qa/generate_answers.yaml

```
@@ -0,0 +1,15 @@
+- role: system
+  content: You are a very knowledgeable AI Assistant that will faithfully assist the user with their task.
+
+- role: user
+  content: |
+    Answer the question based on the provided document.  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)